src/components/form-helpers/form-helpers.js

```diff
@@ -319,9 +322,9 @@ const _PhoneFieldItem = ({
       autoPlaceholder: 'aggressive',
       preferredCountries: ['ar', 'mx', 'co', 'es', 'ec', 'cl', 'pe', 'us'],
       initialCountry: 'auto',
-      geoIpLookup: (callback) => {
-        // TODO: determine current country using geolocation
-        callback('ar');
+      geoIpLookup: async (success) => {
+        const countryCode = await ipinfoClient.getCountryCode();
```

Comment:
what happen is fail the request?

Reply:
The request never fails, if ipinfo api returns something wrong by defautl I will return 'AR'

Reply:
> what happen is fail the request?

It never fails